src/components/DatasetDropdown/styles.module.css

```diff
@@ -24,8 +24,13 @@
   z-index: 1;
 }
 
+.dropdownArrow {
+  margin-right: 7px;
```

Comment:
We might wanna make this `.rem` for consistency with other values

Reply:
done